src/NuGet.Core/NuGet.Common/AuthTypeFilteredCredentials.cs

```diff
@@ -3,45 +3,44 @@
 
 using System;
 using System.Collections.Generic;
-using System.Net;
 using System.Linq;
+using System.Net;
 
-
-namespace NuGet.Credentials
+namespace NuGet.Common
 {
     /// <summary>
-    /// Wraps another ICredentials object by returning null if the authType is not in a specified allow list
+    /// Wraps a NetworkCredentials object by returning null if the authType is not in a specified allow list
     /// </summary>
     public class AuthTypeFilteredCredentials : ICredentials
     {
-        private readonly IList<string> _authTypes;
-        private readonly ICredentials _innerCredential;
+        public IReadOnlyList<string> AuthTypes { get; }
```

Comment:
Why remove readonly?

Reply:
Turned it into a read-only property instead of a readonly field. The backing field of a read-only auto-property is readonly.